extensions/java/language-configuration.json

```diff
@@ -29,5 +29,70 @@
 			"start": "^\\s*//\\s*(?:(?:#?region\\b)|(?:<editor-fold\\b))",
 			"end": "^\\s*//\\s*(?:(?:#?endregion\\b)|(?:</editor-fold>))"
 		}
-	}
+	},
+		],
+	"onEnterRules": [
+		{
+			"beforeText": {
```

Comment:
Most of the javascript `onEnterRules` have a comment explaining them. Can you also copy over the comments?

Reply:
I don't think any rename is needed for that. The javascript configuration isn't named jsonc. https://github.com/mickaelistria/vscode/blob/9f8a4372d4fad4f57e801124015e4fe4af6583c1/extensions/javascript/javascript-language-configuration.json

Reply:
OK, that's unusual, but whatever works ;)